.github/workflows/shellcheck.yaml

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: Shellcheck
+
+on:
+  push:
+    branches:
+      - "main"
+      - "release-**"
+    paths:
+      - '**.sh'
+  pull_request:
+    branches:
+      - "main"
+      - "release-**"
+    paths:
+      - '**.sh'
+
+permissions:
```

Comment:
```suggestion
env:
  LC_ALL: en_US.UTF-8

defaults:
  run:
    shell: bash


permissions:
```

Reply:
Why do we need?

```
defaults:
	  run:
	    shell: bash
```

Reply:
Actually, why do we need all the suggestions? Thanks!